src/main/sensors/barometer.c

```diff
@@ -260,9 +259,9 @@ uint32_t baroUpdate(void)
         case BAROMETER_NEEDS_CALCULATION:
             baro.dev.get_up();
             baro.dev.start_ut();
-            baro.dev.calculate(&baroPressure, &baro.baroTemperature);
+            baro.dev.calculate(&baro.baroPressure, &baro.baroTemperature);
```

Comment:
This changes behavior of baro code. Was is tested to not break things on other boards?